### PR TITLE
Created branch to add Modals to Portfolio

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### I have created this branch to add modals to my portfolio project. I and my partner worked on some things in my project which are:

✔️ Added Modals for Portfolio
:heavy_check_mark: JavaScript to add modals and add functionality (JS)
:heavy_check_mark: Update markup to make it dynamic (HTML)
:heavy_check_mark: Added styling for Modals (CSS)
:heavy_check_mark: Followed Linters
 :heavy_check_mark: Documented work in a professional way

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,5 +59,70 @@ function closeMain() {
...
+      <p class="project_description">${array[i].longDescription}</p>
+      <ul class="tech_list">
+          <li>${array[i].technologies[0]}</li>
+          <li class="margin_li">${array[i].technologies[1]}</li>
+          <li class="margin_li">${array[i].technologies[2]}</li>
+      </ul>
+      <div class="buttons">
+          <a class="model_btn live" href=${array[i].live_version}>See live<img class="_icon" src="./images/Icon.svg" alt="live icon"></a>
+          <a class="model_btn source" href=${array[i].source_link}>See project<img class="_icon" src="./images/Vector.svg" alt="github icon"></a>
+      </div>
```
Comment (oloomoses) [inline app.js]: - [ ] On the mobile design popup, there is a light border between the tags and the button. This is not reflected in your design. Kindly implement it and adjust the margins accordingly. Check the screenshot below.
- Your design: 
Screenshot from 2021-12-16 13-44-47
- Given design:
Screenshot from 2021-12-16 13-44-09

File: index.html [hunk 2]
```diff
@@ -25,8 +15,16 @@
...
- 
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+        gtag('js', new Date());
+
+        gtag('config', 'G-MZLCHLPPZL');
```
Comment (oloomoses) [inline index.html]: - [ ] Keep all the JavaScript code in JS files, not in the script tag as per the [JavaScript requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (HAMMAS-SALEEM) [reply index.html]: Thanks, sir for approaching me. I would like to say thanks for giving me a thorough review. I would say respectfully that I don't know about this requirement and all my partners including the morning session team and standup team did this the same way. But if you force me to do it so can you give me guidance about using google analytics from an external javascript file.

I'll be thankful to you.

**Comment by HAMMAS-SALEEM** [comment]
Thanks, sir for approaching me. I would like to say thanks for giving me a
thorough review. I would say respectfully that I don't know about this
requirement(using google analytics from an external file) and all my
partners including the morning session team and standup team did this the
same way. But if you force me to do it so can you give me guidance about
using google analytics from an external javascript file.

I'll be thankful to you

**Review by brytebee — APPROVED** [review]
Hi @HAMMAS-SALEEM 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**